src/driver/odoc_unit.ml

```diff
@@ -74,6 +74,8 @@ let pp_index fmt x =
 type 'a t = {
   parent_id : Odoc.Id.t;
   input_file : Fpath.t;
+  input_copy : Fpath.t option;
+      (* Used to stash cmtis from virtual libraries into the odoc dir for voodoo mode *)
```

Comment:
Would adding a link to this PR allow for more information while not making it too verbose?

(When you read something like that you might wonder why we need to "stash" those cmti there, and it's definitely possible to find the PR from the commit from a git blame but is not the best experience!)

Reply:
Hmm, yeah we could do this for now, then when we've got some actual docs we can point there instead.